tests/models/test_transform.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from alitra.models import Transform, Translation
+
+
+def test_transform(default_rotation, robot_frame, asset_frame):
+    with pytest.raises(ValueError):
+        translation = Translation(1, 1, from_=robot_frame, to_=asset_frame)
+        Transform(
+            translation, rotation=default_rotation, from_=asset_frame, to_=robot_frame
+        )
+
+
+def test_transform_without_rotation(robot_frame, asset_frame):
+    with pytest.raises(ValueError):
+        translation = Translation(1, 1, from_=robot_frame, to_=asset_frame)
+        Transform(translation=translation, from_=asset_frame, to_=robot_frame)
```

Comment:
Testing when things do not work is fine, but we should also test a case where the `.from_auler_array()` and `.from_quat_array()` methods work.

Reply:
Never mind, I see that my requested tests are available in test_transformations.py further down